src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbStreamsBySharedTable.java

```diff
@@ -81,6 +92,9 @@ private MtRecord mapRecord(Function<Map<String, AttributeValue>, FieldValue> fie
         FieldValue fieldValue = fieldValueFunction.apply(record.getDynamodb().getKeys());
         MtAmazonDynamoDbContextProvider mtContext = mtDynamoDb.getMtContext();
         // execute in record tenant context to get table mapping
+        System.out.println(
+            fieldValue.getMtContext() + " " + fieldValue.getTableIndex() + " " + fieldValue.getUnqualifiedValue());
+
```

Comment:
probably want to take this out (sorry, forgot to remove it)

Reply:
removed https://github.com/salesforce/mt-dynamo/pull/142/commits/81a062ac8d70516e5729036853ce5dbd380fdf5c